src/library_syscall.js

```diff
@@ -1201,13 +1201,10 @@ function wrapSyscallFunction(x, library, isWasi) {
   }
   post = handler + post;
 
-  if (pre) {
-    var bodyStart = t.indexOf('{') + 1;
-    t = t.substring(0, bodyStart) + pre + t.substring(bodyStart);
-  }
-  if (post) {
-    var bodyEnd = t.lastIndexOf('}');
-    t = t.substring(0, bodyEnd) + post + t.substring(bodyEnd);
```

Comment:
unrelated trivial refactor? seems trivial so sgtm to leave it in, just checking.

Reply:
No.. the old code breaks because we have a `{` which is in a comment.. but `modifyFunction` can handle it... took me a while to find that one!

Reply:
Oh cool, good find + fix!